MAINTENANCE.md

```diff
@@ -0,0 +1,75 @@
+# Issue Tsar Role
```

Comment:
I feel it is time to rename the role to something meaningful such as `Issue supervisor`. This would help enforce the difference in how this role is taken now that it is a more serious position.

Reply:
Yes, I think we need to bring it up at the next Dotty meeting to receive everyone's input about the proper name.